app/core/Engine.ts

```diff
@@ -848,10 +929,21 @@ class Engine {
               origin,
             );
           },
+          getAllowedKeyringMethods: (origin: string) =>
+            keyringSnapPermissionsBuilder(origin),
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          hasPermission: (origin: string, target: any) =>
```

Comment:
Shouldn't this be `string` instead of `any` here? Looking at: https://github.com/MetaMask/core/blob/main/packages/permission-controller/src/PermissionController.ts#L824-L828

```suggestion
          hasPermission: (origin: string, target: string) =>
```

Reply:
good catch